cmd/rook/ceph/cleanup.go

```diff
@@ -58,21 +62,59 @@ func startCleanUp(cmd *cobra.Command, args []string) error {
 			logger.Infof("successfully cleaned up %q directory", cleanupDirPath)
 		}
 		// Remove all the mon directories.
-		monDirs, err := filepath.Glob(path.Join(dataDirHostPath, "mon-*"))
+		if err := cleanMonDir(); err != nil {
+			logger.Errorf("failed to clean up the mon directories in the dataDirHostPath %q", dataDirHostPath)
```

Comment:
also need to log the `err` otherwise we don't know what failed

Reply:
Actually it looks like most errors are just logged and not returned inside that function. How about if that function doesn't return any errors? It could just log its errors directly.